vm/src/obj/objbyteinner.rs

```diff
@@ -319,6 +474,18 @@ impl PyByteInner {
         new
     }
 
+    pub fn swapcase(&self, _vm: &VirtualMachine) -> Vec<u8> {
+        let mut new: Vec<u8> = Vec::new();
```

Comment:
You know the size of the result vector so you should initialize the vector with that size.

Reply:
done